cf_units/tests/test_unit.py

```diff
@@ -373,8 +373,13 @@ def test_multiply_not_numeric(self):
     def test_multiply_with_unknown_unit(self):
         u = Unit('unknown')
         v = Unit('meters')
+        self.assertIsNotNone
         self.assertTrue((u * v).is_unknown())
         self.assertTrue((v * u).is_unknown())
+        self.assertIsNotNone((u * v).origin)
+        self.assertIsNotNone((v * u).origin)
+        self.assertIsNotNone((u * v).ut_unit)
+        self.assertIsNotNone((v * u).ut_unit)
```

Comment:
@pdearnshaw You could be stricter here and check for `_ud. NULL_UNIT`

Reply:
@pdearnshaw Can you do the same test extension for `Test_divide.test_divide_with_unknown_unit`, thanks. 

This is the other operator to call `Unit._op_common`, and it'll give us consistent test coverage for your use case.

Reply:
I have changed this to assertEqual(..., unit._ud.NULL_UNIT) as recommended and replicated the tests for the divide operator.